addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/handler/AirQualityHandler.java

```diff
@@ -266,7 +266,7 @@ private AirQualityJsonResponse getAirQualityData(String location, Integer statio
         }
 
         // Update the thing status
-        if (resultOk && result != null) {
+        if (resultOk) {
```

Comment:
resultOk will only be true if result != null && result.getData() != null && result.getStatus() != "error", see line 248